geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ClientDataSerializerMessage.java

```diff
@@ -223,8 +223,9 @@ public boolean needsNoAuthorizationCheck() {
   @Override
   public String toString() {
     StringBuffer buffer = new StringBuffer();
```

Comment:
Might as well change these toString methods to use StringBuilder instead of StringBuffer. StringBuffer is fully synchronized.